src/idea.js

```diff
@@ -6,7 +6,21 @@ class Idea {
     this.star = false;
   }
   saveToStorage() {
-    // storage.push(this)
-    // localStorage.setItem(this.id, this)
+    if (localStorage.getItem("ideas") === null){
+      var ideas = {};
+    } else {
+     var ideas = JSON.parse(localStorage.getItem("ideas"));
+    }
```

Comment:
We've got this if-else statement in two places, we might be able to refactor it down to one.